src/osemgrep/cli_scan/Scan_CLI.ml

```diff
@@ -832,16 +839,16 @@ let cmdline_term ~allow_empty_config : conf Term.t =
    * of the corresponding '$ o_xx $' further below! *)
   let combine allow_untrusted_validators autofix baseline_commit common config
       dataflow_traces diff_depth dryrun dump_ast dump_command_for_core
-      dump_engine_path emacs error exclude_ exclude_rule_ids force_color
-      gitlab_sast gitlab_secrets include_ incremental_output json junit_xml lang
-      ls matching_explanations max_chars_per_line max_lines_per_finding
-      max_memory_mb max_target_bytes metrics num_jobs no_secrets_validation
-      nosem optimizations oss output pattern pro project_root pro_intrafile
-      pro_lang remote replacement respect_gitignore rewrite_rule_ids sarif
-      scan_unknown_extensions secrets severity show_supported_languages strict
-      target_roots test test_ignore_todo text time_flag timeout
-      _timeout_interfileTODO timeout_threshold trace validate version
-      version_check vim =
+      dump_engine_path emacs error exclude_ exclude_rule_ids files_only
+      force_color gitlab_sast gitlab_secrets include_ incremental_output json
+      junit_xml lang ls matching_explanations max_chars_per_line
+      max_lines_per_finding max_memory_mb max_target_bytes metrics num_jobs
+      no_secrets_validation nosem optimizations oss output pattern pro
+      project_root pro_intrafile pro_lang remote replacement respect_gitignore
+      rewrite_rule_ids sarif scan_unknown_extensions secrets severity
+      show_supported_languages strict target_roots test test_ignore_todo text
+      time_flag timeout _timeout_interfileTODO timeout_threshold trace validate
+      version version_check vim =
```

Comment:
Should we perhaps commit to having these all be on separate lines so diffs are a bit better?